README.md

```diff
@@ -1 +1,22 @@
 # Python API for interacting with FLOCX
+
+Overview
+--------
+
+This is a client for the OpenStack `Market API
+It provides:
+
+* a command-line interface: ``openstack market``
+
+``python-flocxclient`` is licensed under the Apache License, Version 2.0,
+like the rest of OpenStack.
+
+
+``openstack market`` CLI
+---------------------------
+
+The ``openstack market`` command line interface is available when the marketplace
+ plugin (included in this package) is used with the `OpenStackClient
+https://docs.openstack.org/python-openstackclient/latest/
+
```

Comment:
Can you add one or two actual example commands here? Eventually we need more substantial API documentation, but something brief here should be enough to start.